conans/test/build_helpers/autotools_configure_test.py

```diff
@@ -473,94 +473,89 @@ def cross_build_command_test(self):
                                                 "compiler.libcxx": "libstdc++"}),
                                   None, runner)
         ab = AutoToolsBuildEnvironment(conanfile)
-        ab.configure()
         if platform.system() == "Windows":
-            # Not crossbuilding
-            self.assertFalse(ab.host)
-            self.assertFalse(ab.build)
```

Comment:
Are not these checks done anymore?

Reply:
No, the command in Windows has no `--host` neither `--build` arguments... I think there is no need to test it here.